pandas/tests/io/excel/test_readers.py

```diff
@@ -1759,3 +1759,53 @@ def test_corrupt_files_closed(self, engine, tmp_excel):
                 pd.ExcelFile(tmp_excel, engine=engine)
             except errors:
                 pass
+
+    def test_mi_header_skiprows1(self, engine, read_ext):
+        if engine is None and read_ext == ".xlsx":
+            with open("test_mi_holes.xlsx", "rb") as f:
+                expected = pd.read_excel(
+                    f, sheet_name="expected", header=[0, 1], index_col=[0, 1]
+                )
```

Comment:
Instead of adding files to the data/ directory, is it possible to generate these sheets on-the-fly? E.g.

```python
df = pd.DataFrame(...)
df.to_excel(tmp_excel, sheet_name="test", index=False)
```

`tmp_excel` here is a pytest fixture.